website/docs/vs-code-snippets/snippets.md

```diff
@@ -10,7 +10,7 @@ We have compiled a collection of useful templates in the form of
 
 To add these snippets to your VS Code: `ctrl+shift+p` > Type 'Configure user
 snippets' > Select `python.json`. All of these snippets are available here:
-[python.json](https://github.com/Azure/azureml-web/blob/main/website/docs/vs-code-snippets/python.json)
+[python.json](https://github.com/Azure/azureml-examples/blob/main/website/docs/vs-code-snippets/python.json)
```

Comment:
looks like this got changed back to examples - should be web.

I updated the python script that generates this page, you should just have to pull and rerun it (Or just swap the url manually...)

Reply:
sent a new PR